<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XConnect</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/XDeck_desktop_none_open.png);</string>
   </property>
   <widget class="QToolButton" name="toolButtonConnect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>76</y>
      <width>117</width>
      <height>85</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/images/XDeck_button_connect.png</normaloff>:/images/images/XDeck_button_connect.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>118</width>
      <height>85</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonDisconnect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>74</y>
      <width>140</width>
      <height>79</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/images/XDeck_button_disconnect.png</normaloff>:/images/images/XDeck_button_disconnect.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>143</width>
      <height>79</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonNetwork">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>10</y>
      <width>190</width>
      <height>56</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/images/XDeck_button_greyed_network.png</normaloff>:/images/images/XDeck_button_greyed_network.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>190</width>
      <height>56</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonSoftware">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>70</y>
      <width>190</width>
      <height>56</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/images/XDeck_button_greyed_software.png</normaloff>:/images/images/XDeck_button_greyed_software.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>190</width>
      <height>56</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frameSoftware">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>280</y>
      <width>373</width>
      <height>458</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/images/XDeck_panel_software.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QToolButton" name="toolButtonBlackMask">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>71</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_black_mask.png</normaloff>:/images/images/XDeck_button_std_black_mask.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonCloak">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>71</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_cloak.png</normaloff>:/images/images/XDeck_button_std_cloak.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonDarcKnight">
     <property name="geometry">
      <rect>
       <x>189</x>
       <y>71</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_darc_knight.png</normaloff>:/images/images/XDeck_button_std_darc_knight.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonFlatline">
     <property name="geometry">
      <rect>
       <x>273</x>
       <y>71</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_flatline.png</normaloff>:/images/images/XDeck_button_std_flatline.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonGenie">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>154</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_genie.png</normaloff>:/images/images/XDeck_button_std_genie.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonGuardDog">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>154</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_guard_dog.png</normaloff>:/images/images/XDeck_button_std_guard_dog.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonHammer">
     <property name="geometry">
      <rect>
       <x>189</x>
       <y>154</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_hammer.png</normaloff>:/images/images/XDeck_button_std_hammer.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonHellhound">
     <property name="geometry">
      <rect>
       <x>273</x>
       <y>154</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_hellhound.png</normaloff>:/images/images/XDeck_button_std_hellhound.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonHydra">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>237</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_hydra.png</normaloff>:/images/images/XDeck_button_std_hydra.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonLightningBug">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>237</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_lightning_bug.png</normaloff>:/images/images/XDeck_button_std_lightning_bug.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonReaper">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>189</x>
       <y>237</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_reaper.png</normaloff>:/images/images/XDeck_button_std_reaper.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonShield">
     <property name="geometry">
      <rect>
       <x>273</x>
       <y>237</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_shield.png</normaloff>:/images/images/XDeck_button_std_shield.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButtonWizardBook">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>320</y>
       <width>80</width>
       <height>79</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/images/XDeck_button_std_wizard_book.png</normaloff>:/images/images/XDeck_button_std_wizard_book.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>387</x>
      <y>260</y>
      <width>608</width>
      <height>477</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/images/XDeck_panel_network.png);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonShutdown">
    <property name="geometry">
     <rect>
      <x>939</x>
      <y>11</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/images/XDeck_button_shutdown.png</normaloff>:/images/images/XDeck_button_shutdown.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>42</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
